Description: Replace LIBDL_LDFLAGS by -ldl
 LIBDL_LDFLAGS is not replaced in tss2-esys.pc. We replace it by hand.
Author: Ying-Chun Liu (PaulLiu) <paulliu@debian.org>
Last-Update: 2019-11-10

Index: tpm2-tss-2.4.0/lib/tss2-esys.pc.in
===================================================================
--- tpm2-tss-2.4.0.orig/lib/tss2-esys.pc.in
+++ tpm2-tss-2.4.0/lib/tss2-esys.pc.in
@@ -10,4 +10,4 @@ Version: @VERSION@
 Requires.private: tss2-mu tss2-sys
 Cflags: -I${includedir}
 Libs: -ltss2-esys -L${libdir}
-Libs.private: @LIBDL_LDFLAGS@ @LIBSOCKET_LDFLAGS@ @TSS2_ESYS_LDFLAGS_CRYPTO@
+Libs.private: -ldl @LIBSOCKET_LDFLAGS@ @TSS2_ESYS_LDFLAGS_CRYPTO@
